assets/js/pages/userHome.js

```diff
@@ -23,8 +24,18 @@ const UserHomepage = ({ history, location }) => {
   useEffect(() => {
     fetchUsers(setUsers);
     fetchTeams(setTeams);
+    fetchTeamss();
   }, []);
 
+  const fetchTeamss = () => {
+    axios
+      .get("http://127.0.0.1:8000/api/teams")
```

Comment:
Note: čia būtų gerai kreiptis į api link. Ne į serverio ip adresą o linką, for example, `https://devCollab.com`. Kaip reiktų testuoti lokaliai? tai tiesiog host faile jums reiktų prisidėti vieną vietą: 
- sudo nano /etc/hosts/
- add this "127.0.0.1:8000 devCollab.com"
- ctrl+o
- ctrl+x
Tačiau manau, kad šiuo metu nereikės. Tačiau ateičiai yra svarbu tai atsminti.
@Karcikovas am I right?

Reply:
sorry, ten taip neturejo but palikta, fe/register pr'e pataisiau

Reply:
Nieko tokio :sweat_smile: tam Pull Request'ai ir skirti :sweat_smile: